app/geo/gui/src/ts/twxplore/gui/geo/reducerFunctions/updateAttributeStatesOfFeatureType.ts

```diff
@@ -9,42 +9,42 @@ export const updateAttributeStatesOfFeatureType = (
   addedFeature: MapFeature
 ) => {
   //Loop through the feature's attribute
-  for (const attribute of Object.keys(addedFeature)) {
+  for (const attributeName of Object.keys(addedFeature)) {
+    const attributeKey = attributeName as keyof MapFeature;
     //If the arttribute is numeric
-    if (getFeatureAttributeByName(attribute).isNumeric) {
+    if (getFeatureAttributeByName(attributeName).isNumeric) {
       //If this is the first time coming across this attribute for this feature type
       if (
-        !attributeStatesOfFeatureType[attribute].min &&
-        !attributeStatesOfFeatureType[attribute].min
+        !attributeStatesOfFeatureType[attributeName].min &&
```

Comment:
Seems like you could put attributeStatesOfFeatureType[attributeName] into a variable, or?

Reply:
Variable named attributeStateOfFeatureType.
Which is awfully close to attributesStatesOfFeatureType[attributeName] but it's an accurate name.